lib/api/apiUtils/object/objectLockHelpers.js

```diff
@@ -0,0 +1,96 @@
+const { errors } = require('arsenal');
+/**
+ * Calculates retain until date for the locked object version
+ * @param {object} retention - includes days or years retention period
+ * @return {object} the date until the object version remains locked
+ */
+function calculateRetainUntilDate(retention) {
+    const { days, years } = retention;
+    const date = new Date();
+    // Calculate the number of days to retain the lock on the object
+    const retainUntilDays = days || years * 365;
```

Comment:
Does this line need parentheses here
```suggestion
    const retainUntilDays = days || (years * 365);
```
Not sure if without them this line might be read as 
```suggestion
    const retainUntilDays = (days || years) * 365;
```

Reply:
![image](https://user-images.githubusercontent.com/8585244/84533215-838aaa80-ac9c-11ea-8ba7-eceaa6a6f124.png)


Seems fine without parenthesis but I had to check

Reply:
yup, conditional evaluation has higher precedence, so at first `days` gets evaluated, if it's not truthy then moves on to right part of `||` and sets the value to `years * 365` but I can add the parentheses still if it'd feel/look safer?

Reply:
nah, it's cool. thanks for verifying